apps/api/src/index.ts

```diff
@@ -138,7 +138,7 @@ app.use(
 
 // Apply rate limiting middleware AFTER authentication
 // This ensures we can properly rate limit by agent/user ID
-app.use(rateLimiterMiddleware);
+//app.use(rateLimiterMiddleware);
```

Comment:
@dtbuchholz as I mentioned the other day, we need a proper way to disable and/or adjust the rate limiter so it doesn't block the local load testing